src/portalbackend/PortalBackend.DBAccess/Repositories/CompanyCertificateRepository.cs

```diff
@@ -99,4 +99,11 @@ sorting switch
                 companyCertificate.ValidTill
                 ))
         .SingleOrDefaultAsync();
+
+    public Task<(byte[] Content, string FileName, MediaTypeId MediaTypeId, bool IsExists, bool IsStatusLocked)> GetCompanyCertificateDocumentDataAsync(Guid documentId, DocumentTypeId documentTypeId) =>
+            _context.Documents
+            .Where(x => x.Id == documentId &&
+                   x.DocumentTypeId == documentTypeId)
+            .Select(x => new ValueTuple<byte[], string, MediaTypeId, bool, bool>(x.DocumentContent, x.DocumentName, x.MediaTypeId, x.Id == documentId, x.DocumentStatusId == DocumentStatusId.LOCKED))
```

Comment:
this can directly be set to true since we're having the filter on matching ids in the where clause. Please change to:

```suggestion
            .Select(x => new ValueTuple<byte[], string, MediaTypeId, bool, bool>(x.DocumentContent, x.DocumentName, x.MediaTypeId, true, x.DocumentStatusId == DocumentStatusId.LOCKED))
```

Reply:
changed required condition